spark/src/test/java/org/apache/iceberg/actions/TestExpireSnapshotsAction.java

```diff
@@ -952,5 +952,19 @@ public void testExpireOlderThanWithRollbackAndMergedManifests() {
 
     checkExpirationResults(1, 1, 1, result);
   }
+
+  @Test
+  public void testExpireOnEmptyTable() {
+
+    Set<String> deletedFiles = Sets.newHashSet();
+
+    // table has no data, testing ExpireSnapshots should not fail with no snapshot
+    ExpireSnapshotsActionResult result = Actions.forTable(table).expireSnapshots()
+            .expireOlderThan(System.currentTimeMillis())
```

Comment:
nit: this should be formatted with 4 spaces instead of 8 like in the rest of the tests